_data/sidebars/documentation.yml

```diff
@@ -1,8 +1,7 @@
 title: Documentation
 subitems:
   - title: Quick start guide
-    title_url: /docs/
-  - title: Getting started with WorkflowHub
+    title_url: /docs/index
```

Comment:
Is this needed?

Usually `.../index` the default rendered when visiting `.../`

Reply:
@fbacall - you're right, we don't need `index` there